ISSUE_TEMPLATE.md

```diff
@@ -4,13 +4,20 @@
 - Discussion Link: <!-- link to SIG mailing list thread, meeting, or recording where the Enhancement was discussed before KEP creation -->
 - Primary contact (assignee):
 - Responsible SIGs:
-- Enhancement target (which target equals to which milestone):
-  - Alpha release target (x.y)
-  - Beta release target (x.y)
-  - Stable release target (x.y)
-- Documentation PR: <!-- link to kubernetes/website PR -->
-  - Alpha:
-  - Beta:
-  - Stable:
+- [ ] Alpha
+  - [ ] Release Target (x.y): <!-- target equals to which milestone -->
+  - [ ] KEP (`k/enhancements`) update PR(s):
+  - [ ] Code (`k/k`) update PR(s):
+  - [ ] Docs (`k/website`) update PR(s):
+- [ ] Beta
```

Comment:
partially feel like we dont need to add the noise of beta and stable until it's time to add them..

Reply:
I agree that it's cleaner that way, but I also think that having it in the issue template serves as a reminder to the Enhancement owners that they eventually should go back and add beta/stable information in the description. No strong opinions either way though.

Reply:
it's been updated to include beta/stable as a comment as suggested by @justaugustus below.